UsageExamples/OoplesIndicatorLibraryExample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Alpaca.Markets;
+using OoplesFinance.StockIndicators.Models;
+using static OoplesFinance.StockIndicators.Calculations;
+
+namespace UsageExamples
+{
+    public class OoplesIndicatorLibraryExample
+    {
+        const string paperApiKey = "REPLACEME";
+        const string paperApiSecret = "REPLACEME";
+        const string symbol = "AAPL";
+
+        public async Task RunExample()
+        {
+            var startDate = new DateTime(2021, 01, 01);
+            var endDate = new DateTime(2021, 12, 31);
+
+            var client = Environments.Paper.GetAlpacaDataClient(new SecretKey(paperApiKey, paperApiSecret));
```

Comment:
Use the `using` statement for properly disposing of the `client` object instance or implement `IDisposable` in the same way as it is done in other examples.